doc/doxyfile.in

```diff
@@ -44,14 +44,14 @@ PROJECT_NUMBER         = @VERSION_MAJOR@.@VERSION_MINOR@.@VERSION_PATCH@
 # for a project that appears at the top of each page and should give viewer a
 # quick idea about the purpose of the project. Keep the description short.
 
-PROJECT_BRIEF          = 
+PROJECT_BRIEF          =
```

Comment:
Editing this file it totally fine. I've last updated the Doxygen file and it was not by just loading and saving it through Doxygen. I saved an empty template, compared it with our previous version and then added/removed settings that were changed/removed/added.